packages/arb-token-bridge-ui/src/components/TransferPanel/TransferDisabledDialog.tsx

```diff
@@ -49,16 +56,41 @@ export function TransferDisabledDialog() {
       onClose={onClose}
     >
       <div className="flex flex-col space-y-4 py-4">
-        <p>
-          Unfortunately, <span className="font-medium">{unsupportedToken}</span>{' '}
-          has a custom bridge solution that is incompatible with the canonical
-          Arbitrum bridge.
-        </p>
-        <p>
-          For more information please contact{' '}
-          <span className="font-medium">{unsupportedToken}</span>
-          &apos;s developer team directly or explore their docs.
-        </p>
+        {/* teleport transfer disabled content if token is not in the allowlist */}
+        {isTeleportMode ? (
+          <>
+            <p>
+              Unfortunately,{' '}
+              <span className="font-medium">{unsupportedToken}</span> is not
+              supported for LayerLeap transfers yet.
+            </p>
+            <p>
+              For more information please contact{' '}
+              <ExternalLink
+                href="https://discord.com/invite/ZpZuw7p"
+                className="underline"
+              >
+                Discord
+              </ExternalLink>{' '}
+              and reach out in #support for assistance..
```

Comment:
i thought we agreed to explain to users that they should deposit from L1->L2 then L2->l3 manually for tokens unsupported by teleporter

why is this one showing?